recipes/slurm_install.rb

```diff
@@ -25,6 +25,18 @@
   action :purge
 end
 
+slurm_build_deps = value_for_platform(
+  'ubuntu' => {
+    'default' => %w(libjson-c-dev libhttp-parser-dev),
+  },
+  'default' => %w(json-c-devel http-parser-devel)
```

Comment:
Please fix the rubocop check by changing 
`%w(libjson-c-dev libhttp-parser-dev),`
to
`['libjson-c-dev', 'libhttp-parser-dev']`
etc